tethys_portal/settings.py

```diff
@@ -104,8 +101,8 @@
 DEFAULT_DB.setdefault('HOST', 'localhost')
 DEFAULT_DB.setdefault('PORT', 5436)
 
-LOGGING_CONFIG = portal_config_settings.pop('LOGGING_CONFIG', {})
-# See https://docs.djangoproject.com/en/1.8/topics/logging/#configuring-logging for more logging configuration options.
+# See https://docs.djangoproject.com/en/3.2/ref/settings/#logging-config for more logging configuration options.
+LOGGING_CONFIG = portal_config_settings.pop('LOGGING_CONFIG', 'logging.config.dictConfig')
```

Comment:
I think we should just remove this completely.

Reply:
Done